src/libs/UserUtils.ts

```diff
@@ -25,36 +27,29 @@ import * as defaultAvatars from '../components/Icon/DefaultAvatars';
  *          }
  *      }
  * }}
- *
- * @param {Object} loginList
- * @param {Object} loginList.errorFields
- * @returns {Boolean}
  */
-function hasLoginListError(loginList) {
-    return _.some(loginList, (login) => _.some(lodashGet(login, 'errorFields', {}), (field) => !_.isEmpty(field)));
+function hasLoginListError(loginList: LoginList): boolean {
+    const errorFields = loginList?.errorFields ?? {};
```

Comment:
Is the type `loginList: LoginList` correct? if it is, we wouldn't need the question mark in `loginList?`

Reply:
Yes it's correct, the thing is that this function is still used in some JS files and wrong values are passed and tests were failing. Once we get to these files that use this function the underlying issue will be fixed.

Either way, it's okay to make code safe and use `nullish coalescing` and `optional chaining` even if types says it's not necessary.

Reply:
> Either way, it's okay to make code safe and use nullish coalescing and optional chaining even if types says it's not necessary.

I disagree with this, I expect to be able to trust our types, but I understand that this may be acceptable while in a transitional period

I'm not sure why we didn't prefer `loginList: Login | undefined | null` if we had cases where we are passing `undefined` or `null` 🤷